app/views/questions/show.html.erb

```diff
@@ -18,8 +18,25 @@
             <div class="center">
               <%= f.submit "Submit answer", class: 'btn btn-theme btn-xl', id: "submit_answer" %>
             </div>
-      <%end%>
-      <div id="response">
+
+           <b><%= link_to "previous", "/questions/#{@prev_question_id}" %></b>    
+          <b><%= link_to "next", "/questions/#{@next_question_id}" %></b>
```

Comment:
This bundle of work doesn't include the `next` and `previous` functionality, so can you remove these lines before we commit it to the main project?